### PR TITLE
feat: remove vendor entry && enable split chunks

### PR DESCRIPTION
**Pull request by shabanov-dlg**

### CODE REVIEW

File: src/web/createWebpackConfig.js [hunk 1]
```diff
@@ -22,8 +22,7 @@ function createWebpackConfig(options: WebOptions) {
     context: resolve(root),
     mode: process.env.NODE_ENV,
     entry: {
-      app: [entry.js, entry.css],
-      vendor: ['@dlghq/dialog-java-core'],
+      app: [entry.js, entry.css]
```
Comment (meafmira) [inline src/web/createWebpackConfig.js]: No trailing comma. Use prettier for all files

File: src/web/configureOptimization.js [hunk 2]
```diff
@@ -9,8 +9,34 @@ const TerserPlugin = require('terser-webpack-plugin');
...
+      cacheGroups: {
+        vendor: {
+          test: /[\\/]node_modules[\\/]/,
+          name(module: { context: string }) {
+            // Splitting of all dependencies into 2 files - "dialog" packages and all the rest
+            const nameData = module.context.match(/[\\/]node_modules[\\/](.*?)([\\/]|$)/);
+
+            let packageName = null;
+            if (nameData && nameData.length > 0) {
+              packageName = nameData[1].replace('@', '')
```
Comment (meafmira) [inline src/web/configureOptimization.js]: No semicolon. Use prettier

File: src/web/createWebpackConfig.js (hunk 1 above)
Comment (shabanov-dlg) [reply src/web/createWebpackConfig.js]: fixed

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dlghq/dialog-scripts",
-  "version": "1.2.10",
+  "version": "1.2.11-split-chunks.0",
```
Comment (meafmira) [inline package.json]: Remove version bump

**Pull request merged**